### ISSUE TITLE
Nadanie przesyłki poprzez wysyłam z allegro Produkcja i Sandbox

### ISSUE THREAD

**Issue by Morph21** [allegro/allegro-api#9226]
Występuje problem: 
 - [x] z zasobem REST API / with a REST API resource
 - [x] ze środowiskiem produkcyjnym / with the production environment
 - [x] ze środowiskiem testowym / with the test environment
 - [ ] z logowaniem lub autoryzacją / with login or authorization
 - [ ] na portalu https://developer.allegro.pl/ / at https://developer.allegro.pl/en/
 - [ ] inny / other

:beetle: **Opis / Description**

Witam, podczas próby nadania paczki klient zgłosił nam problem że dostaje zwrotną informację z błędem.
"Wybrany punkt nadania nie jest obsługiwany przez tego przewoźnika"

Próbowaliśmy powtórzyć błąd na środowisku Sandbox, także się pojawia.

Przewoźnik: Allegro International Automaty Paczkowe Czechy
Punkt nadania: GDA26M

**Comment by MaciejFrackowiak** [comment]
Proszę podaj identyfikator zamówienia przesyłki lub trace-id wywołania.

**Comment by Morph21** [comment]
To dostajemy w moment próby utworzenia Przesyłki
trace-id=[303d635bd642a5c5]

A to dostajemy w moment sprawdzenia statusu utworzenia przesyłki
trace-id=[3536c11a4747df6f]

**Comment by MaciejFrackowiak** [comment]
Nie wskazuj konkretnego punktu nadania przesyłki w przesyłanym requeście.

**Comment by Aleksandrze** [comment]
> Nie wskazuj konkretnego punktu nadania przesyłki w przesyłanym requeście.

Czy istnieje jakiś sposób na określenie czy przewoźnik wymaga podania punktu nadania (sender.point) lub nie?

W dokumentacji jest napisane że sender.point “wymagane, jeśli adresem nadawczym jest punkt odbioru” np. przesyłki wysłane za pomocą "Poczta Polska" zawszę wymaga podania identyfikatora placówki

https://developer.allegro.pl/tutorials/jak-zarzadzac-przesylkami-przez-wysylam-z-allegro-LRVjK7K21sY#jak-utworzyc-nowa-paczke

**Comment by MaciejFrackowiak** [comment]
@Aleksandrze Nie, jest to specyficzne dla każdego operatora logistycznego. W przypadku InPost takiego punktu jednak nie należy wskazywać.

**Comment by Morph21** [comment]
Czy w takim razie możemy dostać listę dla których przewoźników należy to przesłać a dla których nie można?

**Comment by MaciejFrackowiak** [comment]
@Morph21 Generalnie dla InPost nie należy wskazywać punktu nadania przesyłki. Tożsamy flow obowiązuje w serwisie WWW - nie wybierasz konkretnego punktu nadania przesyłki.

**Comment by Morph21** [comment]
@MaciejFrackowiak ja to rozumiem, nie mniej jednak było by miło jakby informacja o tym dla którego przewoźnika trzeba to podać była gdzieś dostępna. Bo z jednej strony w dokumentacji jest napisane że sender point jest wymagany jeżeli jest punkt dostawy a z drugiej strony w api zwracacie błąd co jest po prostu niespójne.

Więc prosiłbym o aktualizacje dokumentacji w tym zakresie

**Comment by Aleksandrze** [comment]
> @Aleksandrze Nie, jest to specyficzne dla każdego operatora logistycznego. W przypadku InPost takiego punktu jednak nie należy wskazywać.

Dziękuje

W takim razie. Czy identyfikator deliveryMethodId dla usług InPost jest taki sam w środowisku produkcyjnym i sandbox? Chodzi mi o identyfikator pobierany z GET /shipment-management/delivery-services, który wynosi: "2488f7b7-5d1c-4d65-b85c-4cbcf253fd93".

**Comment by MaciejFrackowiak** [comment]
@Aleksandrze Nie, każde środowisko ma niezależne od siebie identyfikatory usług.

**Comment by Aleksandrze** [comment]
@MaciejFrackowiak W takiej przypadku, jeśli to możliwe, proszę o podanie identyfikatora InPost z produkcji.

**Comment by MaciejFrackowiak** [comment]
@Aleksandrze Odczytasz go korzystając ze wspomnianego [GET /shipment-management/delivery-services](https://developer.allegro.pl/documentation/#operation/getDeliveryServices).

**Comment by MaciejFrackowiak** [comment]
@Morph21 Niiestety, wymagania operatorów logistycznych oraz dostawców usług rządzą się swoimi "prawami", które są na tyle specyficzne, że nie sposób nam umieścić osobnych zapisów per każdy operator w dokumentacji. Dlatego z pomocą przychodzi tu Sandbox i proces, który można ręcznie "przeklikać" w oparciu o serwis WWW - daje to jednoznaczne odpowiedzi. Również co kluczowe i istotne - wymagania operatorów nie zmieniają się ot tak, więc można przyjąć, że dany flow będzie obowiązywać przez dłuższy okres czasu.

**Comment by Morph21** [comment]
@MaciejFrackowiak ok, dziękuję za pomoc
